nupic/encoders/multi.capnp

```diff
@@ -0,0 +1,51 @@
+@0xf07c1f24bf12fe13;
+
+using import "adaptivescalar.capnp".AdaptiveScalarEncoderProto;
+using import "category.capnp".CategoryEncoderProto;
+using import "coordinate.capnp".CoordinateEncoderProto;
+using import "date.capnp".DateEncoderProto;
+using import "delta.capnp".DeltaEncoderProto;
+using import "geospatial_coordinate.capnp".GeospatialCoordinateEncoderProto;
+using import "log.capnp".LogEncoderProto;
+using import "pass_through.capnp".PassThroughEncoderProto;
+using import "random_distributed_scalar.capnp".RandomDistributedScalarEncoderProto;
+using import "scalar.capnp".ScalarEncoderProto;
+using import "sdrcategory.capnp".SDRCategoryEncoderProto;
+using import "sparse_pass_through_encoder.capnp".SparsePassThroughEncoderProto;
+
+# Next ID: 14
+struct MultiEncoderProto {
+  struct IntType {
+    value @0 :UInt32;
+  }
+
+  # Next ID: 4
+  struct EncoderDetails(Index, Name, Encoder, Offset) {
```

Comment:
Can you merge this with `EncoderUnion`? Since only one encoder type can be present in the union, we don't need to store an index, name, and offset for each one.